test/compute-engine/simplify.test.ts

```diff
@@ -1160,64 +960,31 @@ const RULES: Rule[] = [
     replace: '\\infty',
     condition: ({ x }) => x.isNegative === true && x.isFinite === true,
   },
-  {
-    match: 'x/y',
-    replace: '\\operatorname{NaN}',
-    condition: ({ x, y }) => x.isInfinity === true && y.isInfinity === true,
-  },
-
   //Infinity and Powers (doesn't work for a=\\pi)
   {
-    match: 'a^\\infty',
-    replace: '\\infty',
-    condition: ({ a }) => a.isGreater(1) === true,
-  },
-  {
-    match: 'a^\\infty',
+    match: '\\infty^a',
     replace: '0',
-    condition: ({ a }) => a.isPositive === true && a.isLess(1) === true,
+    condition: ({ a }) => a.isNegative === true,
   },
   {
     match: '\\infty^a',
-    replace: '0',
-    condition: ({ a }) => a.isNegative === true,
+    replace: '\\infty',
+    condition: ({ a }) => a.isPositive === true,
   },
   {
     match: '(-\\infty)^a',
     replace: '0',
     condition: ({ a }) => a.isNegative === true,
   },
-  {
-    match: 'a^{-\\infty}',
-    replace: '0',
-    condition: ({ a }) => a.isGreater(1) === true,
-  },
-  {
-    match: 'a^{-\\infty}',
-    replace: '\\infty',
-    condition: ({ a }) => a.isPositive === true && a.isLess(1) === true,
-  },
   //This one works for \\pi
   // {match:'\\infty^a',replace:'\\infty',condition:id=>id._a.isPositive===true},
 
-  //Logs and Infinity
-  '\\ln(\\infty) -> \\infty',
-  {
-    match: '\\log_c(\\infty)',
-    replace: '\\infty',
-    condition: ({ c }) => c.isGreater(1) === true,
-  },
+  //@fixme
```

Comment:
What is the problem with this rule?

Reply:
It does it by default. Neither or nor the default works for $e^\infty$ because it doesn't realize that $e>1$

Reply:
OK